asmrun/startup.c

```diff
@@ -86,7 +86,7 @@ extern value caml_start_program (void);
 extern void caml_init_ieee_floats (void);
 extern void caml_init_signals (void);
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && __STDC_SECURE_LIB__ >= 200411L
```

Comment:
Ah - good find! I was going to say that in the absence of documentation confirming the behaviour, it would be better to change it!